site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,96 @@
+
+In addition to themes, there is a set of base styles provided by `@dojo/widgets` containing basic utility classes like the screen reader-accessible `visuallyHidden` style. To use it, import `baseCss` separately like this:
+
+```ts
+import * as css from ‘path/to/your/css’;
+import * as baseCss from ‘@dojo/widgets/common/styles/base.m.css’;
+
+@theme(css)
+class MyWidget extends WidgetBase {
+    render() {
+        return v(‘div’, {
+            classes: this.classes().fixed(baseCss.visuallyHidden)
+        }, [ ‘Screen reader instructions’ ]);
+    }
+}
+```
+
+This CSS class will hide a widget visually while allowing it to remain visible to screen readers.
+
+## Writing custom widgets
+
+All DOM attributes can be set with `VirtualDomProperties` so, apart from managing focus, no extra tools are required to create highly accessible widgets. For example, the following code would create an accessible popup button:
+
+```ts
+v('button', {
+    key: 'buttonKey',
+    'aria-controls': 'popupId',
+    'aria-expanded': String(this.properties.expanded),
+    'aria-haspopup': 'true',
+    id: 'buttonId',
+    onclick: this._togglePopup
+}, [ 'Click to open popup' ])
+```
+
+Since this example uses a `<button>` element, it requires neither an explicit `tabindex` nor an `onkeydown` event to work properly with a keyboard. Wherever possible, Dojo 2 takes advantage of built-in accessibility by using native elements. However, while native functionality is the most straightforward path to good accessibility, there are times when it is not possible, such as when no built-in element gives the desired functionality.
+
+For those times, this checklist is a good starting point to ensure that a widget is accessible:
+
+- Ensure nodes that support any sort of user interaction are both focusable and have accessible descriptive text.
+- Leave native focus styles intact or provide a sufficiently visible alternative.
+- Check the [WAI-ARIA Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices-1.1) for information on node attributes and keyboard interaction for many common widget patterns. Even for more complex widgets, these serve as good building blocks.
+- Make a quick proof-of-concept and try [inspecting its accessibility attributes](http://khan.github.io/tota11y/) in the browser and running through one of the [many](https://medium.com/@addyosmani/accessible-ui-components-for-the-web-39e727101a67) [excellent](http://www-03.ibm.com/able/guidelines/ci162/accessibility_checklist.html) [a11y](https://www.wuhcag.com/wcag-checklist/) [checklists](https://ebay.gitbooks.io/oatmeal/content/).
+- Try operating the widget with only a keyboard and, if possible, a screen reader.
+- If applicable, try turning off your sound.
+- Look at your widget with high contrast mode enabled
```

Comment:
When in doubt, https://github.com/theintern/intern-a11y/graphs/contributors ... would be @jason0x43.